--- /dev/null
+++ ../src/FileFinder/FileFinder.php
@@ -53,6 +53,10 @@
             ->notName(['index.php', 'cli-script.php'])
             ->notPath([...$this->getExcludedDirectories(), 'vendor']);
 
+        if ($this->getVendorModuleDirectories() === []) {
+            return $app;
+        }
+
         $vendor = Finder::create()
             ->in($this->getVendorModuleDirectories())
             ->files()
@@ -82,6 +86,7 @@
      */
     public function getAppRootDirectory(): string
     {
+        return '/app';
         foreach (InstalledVersions::getAllRawData() as $data) {
             if (!isset($data['versions']['silverstripe/framework'])) {
                 continue;
@@ -136,6 +141,12 @@
             return $this->excludedDirectories;
         }
 
+        $inDirs = [...$this->getVendorModuleDirectories(), ...$this->getAppDirectories()];
+        
+        if ($inDirs === []) {
+            return [];
+        }
+
         $hits = Finder::create()
             ->in([
                 ...$this->getVendorModuleDirectories(),
@@ -205,6 +216,10 @@
     {
         if ($this->vendorModuleDirectories !== null) {
             return $this->vendorModuleDirectories;
+        }
+
+        if ($this->getVendorModuleRootDirectories() === []) {
+            return [];
         }
 
         $hits = Finder::create()
