--- /dev/null
+++ ../src/Testing/PHPUnit/AbstractRectorTestCase.php
@@ -135,6 +135,8 @@
         }
         // write temp file
         FileSystem::write($inputFilePath, $inputFileContents, null);
+        $classManifest = self::getContainer()->has('Cambis\Silverstan\ClassManifest\ClassManifest') ? $this->make('Cambis\Silverstan\ClassManifest\ClassManifest') : null;
+        \Cambis\SilverstripeRector\Testing\Fixture\FixtureManifestUpdater::addInputFileToClassManifest($inputFilePath, $classManifest);
         $this->doTestFileMatchesExpectedContent($inputFilePath, $inputFileContents, $expectedFileContents, $fixtureFilePath);
     }
     private function forgetRectorsRules() : void
