--- /dev/null
+++ ../src/RuleCodeSamplePrinter/RectorRuleCodeSamplePrinter.php
@@ -10,7 +10,6 @@
 use Symplify\RuleDocGenerator\Printer\MarkdownCodeWrapper;
 use Symplify\RuleDocGenerator\RuleCodeSamplePrinter\ConfiguredRuleCustomPrinter\RectorConfigConfiguredRuleCustomPrinter;
 use Symplify\RuleDocGenerator\ValueObject\CodeSample\ComposerJsonAwareCodeSample;
-use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
 use Symplify\RuleDocGenerator\ValueObject\CodeSample\ExtraFileCodeSample;
 use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
 
@@ -40,14 +39,6 @@
 
         if ($codeSample instanceof ComposerJsonAwareCodeSample) {
             return $this->printComposerJsonAwareCodeSample($codeSample);
-        }
-
-        if ($codeSample instanceof ConfiguredCodeSample) {
-            return $this->configuredCodeSamplerPrinter->printConfiguredCodeSample(
-                $ruleDefinition,
-                $codeSample,
-                $this->rectorConfigConfiguredRuleCustomPrinter
-            );
         }
 
         return $this->diffCodeSamplePrinter->print($codeSample);
