--- /dev/null
+++ ../src/Testing/PHPUnit/AbstractRectorTestCase.php
@@ -3,6 +3,7 @@
 declare (strict_types=1);
 namespace Rector\Testing\PHPUnit;
 
+use Cambis\SilverstripeRector\Testing\Fixture\FixtureManifestUpdater;
 use RectorPrefix202408\Illuminate\Container\RewindableGenerator;
 use Iterator;
 use RectorPrefix202408\Nette\Utils\FileSystem;
@@ -135,6 +136,7 @@
         }
         // write temp file
         FileSystem::write($inputFilePath, $inputFileContents, null);
+        FixtureManifestUpdater::addInputFileToClassManifest($inputFilePath);
         $this->doTestFileMatchesExpectedContent($inputFilePath, $inputFileContents, $expectedFileContents, $fixtureFilePath);
     }
     private function forgetRectorsRules() : void
